<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kuninstaller</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/logo.png</normaloff>icons/logo.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#00aaff;&quot;&gt;Uninstall program&lt;/span&gt;&lt;/p&gt;&lt;p&gt;To uninstall a program, select it from the list and then right-click and select Uninstall or About.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="horizontalSpacer">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>563</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QWidget" name="widget" native="true">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="2">
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>71</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>81</width>
           <height>71</height>
          </size>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string>Current program</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <italic>true</italic>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
         <property name="text">
          <string>Total size: x MB</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="2" rowspan="2">
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>251</width>
        <height>32</height>
       </size>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="placeholderText">
       <string>Search programs ...</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionQuit"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Sett&amp;ings</string>
    </property>
    <addaction name="actionKuninstaller_settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Kuninstaller"/>
    <addaction name="actionAbout_KDE"/>
   </widget>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>P&amp;rogram</string>
    </property>
    <addaction name="actionUninstall"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProgram"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionKuninstaller_settings">
   <property name="icon">
    <iconset theme="settings-configure">
     <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>&amp;Kuninstaller settings ...</string>
   </property>
  </action>
  <action name="actionAbout_Kuninstaller">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;About Kuninstaller</string>
   </property>
  </action>
  <action name="actionAbout_KDE">
   <property name="icon">
    <iconset theme="kde">
     <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>About &amp;KDE</string>
   </property>
  </action>
  <action name="actionUninstall">
   <property name="icon">
    <iconset theme="package-remove">
     <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
   </property>
   <property name="text">
    <string>&amp;Uninstall</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;About this package</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
